templates/zerver/help/desktop-app-install-guide.md

```diff
@@ -46,7 +46,7 @@ If you have Homebrew installed and prefer to use it, here's how.
 4. The app will NOT update automatically. You can repeat these steps
    to upgrade to future versions. <!-- TODO fact check -->
 
-## Installing on Linux
+## [Installing on Linux](#installing-on-linux)
```

Comment:
Maybe worth adding comments above these headings that they're used in the `/apps` page and shouldn't be changed without updating the links.